entrypoint.sh

```diff
@@ -579,6 +580,24 @@ function start_rgw {
   fi
 }
 
+function create_rgw_user {
+
+  # Check to see if our RGW has been initialized
+  if [ ! -e /var/lib/ceph/radosgw/keyring ]; then
+    echo "ERROR- /var/lib/ceph/radosgw/keyring must exist. You can extract it from your current monitor by running 'ceph auth get client.bootstrap-rgw -o /var/lib/ceph/bootstrap-rgw/${CLUSTER}.keyring'"
+    exit 1
+  fi
+
+  mkdir -p "/var/lib/ceph/radosgw/${RGW_NAME}"
+  mv /var/lib/ceph/radosgw/keyring /var/lib/ceph/radosgw/${RGW_NAME}/keyring
+
+  if [ -z "${RGW_USER_SECRET_KEY}" ]; then
+    radosgw-admin user create --uid=${RGW_USER} --display-name="RGW ${RGW_USER} User" -c /etc/ceph/${CLUSTER}.conf
```

Comment:
By default, this command uses `client.admin`, I'm not sure if the bootstrap key is sufficient to create the user.
I'm sure you've tested this already, but I'm wondering if we should not be checking for the admin key instead.

What do you think?